src/extra/canvas.rs

```diff
@@ -24,7 +24,7 @@ pub trait Window<R: Resources>: Output<R> {
     fn swap_buffers(&mut self);
 }
 
-/// A canvas with everything you need to draw on it.
+/// DEPRECATED
 pub struct Canvas<W, D: Device, F> {
```

Comment:
I was under impression that this attribute is only reserved for standard libraries. Will try now.